selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestGitHubRepository.java

```diff
@@ -115,6 +127,21 @@ private void createFile(Path pathToRootContentDirectory, Path pathToFile) throws
     }
   }
 
+  /**
+   * Change content of the file
+   *
+   * @param pathToFile path to specified file
+   * @param content content to change
+   * @throws IOException
+   */
+  public void changeFileContent(String pathToFile, String content) throws IOException {
+    ghRepo.getFileContent(String.format("/%s", pathToFile)).update(content, "change - " + content);
```

Comment:
Adding content to the commit message is not a good idea. Also, it's better to have unified commit message like: `Change file <pathToFile>`